src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -156,11 +135,10 @@ monodroid_typemap_java_to_managed (const char *java)
 	return nullptr;
 }
 
-MONO_API const char *
-monodroid_typemap_managed_to_java (const char *managed)
+inline const char*
```

Comment:
You're inlining a method with a `for` loop?!

Why?

Reply:
To make it faster without incurring another call overhead?